src/locales/skSk.js

```diff
@@ -344,7 +344,7 @@ export default {
     "saintVitusMartyr": "Svätého Víta, mučeníka",  // austria, czechRepublic, germany, switzerland
```

Comment:
@tukusejssirs do you think we could remove theses comments here in this file? I think it's not really their place. And if needed, you can still find them in the code history. Also I would like to transform theses files to `json`, so we are not tempted to add logic or variables here (and `json` unfortunately don't like comments). And it might be also a step to moving localisation and calendars forward, to make theses ressources more independant from the core codebase, so making them easiest to manage and update.

Reply:
Ok thank you. I'll do it in a separated PR (the `.js` -> `.json` files), and we will discuss about the localisation elsewhere in the issue tracker. I'm also thinking about a separated repo containing calendars and localisations, but I need to dive into it first and do some code/architecture experiments (particularly for the calendars). But it was just a side comment andiIn the meantime we need to merge that PR 😄

Reply:
Okay to everything you said. Still I would note that there is a need to have historical calendars (at the Extra-Ordinary Calendar, which is not a _true_ historical calendar; see #23, #51).